plutus-playground/deployment/terraform/templates/nixops_configuration.nix

```diff
@@ -0,0 +1,35 @@
+{ config, pkgs, ... }:
+{
+    imports = [ <nixpkgs/nixos/modules/virtualisation/amazon-image.nix> ];
```

Comment:
Again seems like there's a bunch of duplication between these and some other ones?

Reply:
ah, this is different, these are cloud-init scripts in the form of terraform templates so they can't easily be modularized.